app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  after_save :update_posts_counter
+
+  private
+
+  def update_posts_counter
+    author.increment!(:posts_counter)
+  end
+
+  def retrieve_recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+end
```

Comment:
- [ ] Good job applying one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) 👍🏾  but the `retrieve_recent_comments` method is not private. When you put this method after the `update_posts_counter` method automatically the `retrieve_recent_comments` become private cause the `update_posts_counter` is private unless you make the ` retrieve_recent_comments` public by adding the `public` word before the method or you move it before the `update_posts_counter` method. Kindly fix that 😁 😁